dashboard/static/styles.css

```
@@ -177,6 +177,27 @@ header {
   border-bottom: 2px solid var(--tab-active-border);
 }
 
+/* ── Sync button (read-only venue refresh) ── */
+.btn-sync {
+  background: rgba(56, 189, 248, 0.12);
+  color: #38bdf8;
+  border: 1px solid rgba(56, 189, 248, 0.35);
+  padding: 8px 14px;
+  border-radius: 8px;
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)